jsk_2016_01_baxter_apc/node_scripts/rbo_segmentation_in_bin_node.py

```diff
@@ -82,8 +82,10 @@ def _callback(self, cloud, img_msg, camera_info):
         # generate a binary image
         self.segmentation()
         try:
-            self.img_pub.publish(self.bridge.cv2_to_imgmsg(
-                    self.predicted_segment, encoding="passthrough"))
+            mask_msg = self.bridge.cv2_to_imgmsg(self.predicted_segment, encoding="passthrough")
+            mask_msg.header.stamp = rospy.Time.now()
```

Comment:
> the process took time, so if i do your suggestion, stamp is not correct

What is correct about the stamp for you?
I think the stamp should be when the original input data is generated. As for this node, it is camera image input.